src/hooks/useTheme.ts

```diff
@@ -1,11 +1,26 @@
 import { useContext } from 'react'
+import Cookie from 'js.cookie'
 import { ThemeContext as StyledThemeContext } from 'styled-components'
 import { useTheme as useNextTheme } from 'next-themes'
 
+const COOKIE_THEME_KEY = 'theme'
+
 const useTheme = () => {
   const { resolvedTheme, setTheme } = useNextTheme()
   const theme = useContext(StyledThemeContext)
-  return { isDark: resolvedTheme === 'dark', theme, setTheme }
+  const getThemeCookie = Cookie.get(COOKIE_THEME_KEY)
```

Comment:
why do we need cookie? afaik we have theme property in local storage

Reply:
Because when click perpetual button, the theme will not same as our side.

Reply:
👍